src/components/Mermaid.tsx

```diff
@@ -23,7 +23,7 @@ export const mermaidLoadContent = (config: unknown) => {
 
 export const Mermaid: FunctionComponent<MermaidProps> = ({ code, config }) => {
 	useEffect(() => {
-		mermaidLoadContent(config);
+		mermaidLoadContent(config as MermaidConfig);
```

Comment:
type assertion ist nicht notwendig, wir können in L7 das `unknown` zu `MermaidConfig` ändern.